src/common/cd-special-req-component.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+<div>
+  <select class="select-box form-control" v-model="specialReqSelect" @blur="onBlur" @focus="onFocus">
+    <option value="" selected data-default disabled>{{ $t('Select an option') }}</option>
+    <option value="no">{{ $t('No') }}</option>
+    <option value="yes">{{ $t('Yes') }}</option>
+  </select>
+  <input v-show="selectYes" class="form-control" v-model="specialReqInput" :placeholder="$t('Type here')" @blur="onBlur" @focus="onFocus"/>
```

Comment:
Could benefit from something more specific than "Type here" like "will be late" or "allergic to kids"

Reply:
How is this? 'Please type any special requirements you have here (ex. Need wheelchair access)'

Reply:
Better :) @conoramurphy does that fit ?

Reply:
Just to keep consistent with how we talk elsewhere I'd go with.

`Please tell us about any special requirements that we might need to know of here (eg. Need wheelchair access)`

Reply:
okeeeeee